meta.go

```diff
@@ -0,0 +1,64 @@
+package spotify
+
+import (
+	"net/url"
+	"strings"
+)
+
+type HREF string
+
+// Get returns the most recent version of the object in it's entirety
+func (h *HREF) Get(api *API, obj interface{}) error {
+	url, err := h.GetURL()
+	if err != nil {
+		return err
+	}
+	path := strings.Replace(url.Path, "/v1", "", 1)
```

Comment:
Isn't it a bad idea to remove the `/v1` here? If they make a v2 API in the future, then this'll make a request to that version, which could break our code.

Reply:
True. Let me rethink how to do this correctly.